lib-injection/sources/requirements.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "https://raw.githubusercontent.com/DataDog/auto_inject/refs/heads/main/preload_go/cmd/library_requirements_tester/testdata/requirements_schema.json",
```

Comment:
Caveat: do we need this file in the release package? In that case, I think we need to include the json in the package data parameter:
https://github.com/DataDog/dd-trace-py/blob/main/setup.py#L565

Reply:
This needs to be in the SSI OCI artifact yes - I'm not familiar with `package_data`, @brettlangdon / @emmettbutler  could you confirm pls? 🤔